app/react/Favorites/index.ts

```diff
@@ -0,0 +1,3 @@
+import { FavoriteBanner } from './components/FavoriteBanner';
+
+export { FavoriteBanner };
```

Comment:
@RafaPolit  what you think about 
`export { FavoriteBanner } from './components/FavoriteBanner';`
instead of the 2 lines, this way its shorter and in some cases avoids object binding problems on circular dependencies.